### ISSUE TITLE
Write Project Canvas for data breach tool

### ISSUE THREAD

**Issue by carapang** [barnardos/data-breach-tool#13]
**Need**

As a team
I want to understand what the needs are for this project 
So that I can get an understanding what to implement and what needs it fulfill

**Description**

Detail about the issue, including:
Create a project canvas based on the template

**Acceptance criteria**
- [x] Discuss with team
- [x] Review and finalize and upload the doc to Google (add link here)

**Comment by carapang** [comment]
@Michael-Richards, have you updated this?

**Comment by Michael-Richards** [comment]
@cara Yep it's at https://drive.google.com/drive/folders/18t6QVQ7jgebETM2LXpsmn3YLBscXr7dY
